### ISSUE TITLE
When updating an item in O2M, the whole update data-chain gets displayed on the item

### ISSUE THREAD

**Issue by LevinGermann** [directus/directus#19236]
### Describe the Bug

When editing an entry in a O2M list, I get a weirdly formatted string as the preview.
I understand this is the structure on how Directus updates/deletes entries behind the scenes, but this looks broken from a user perspective.

https://github.com/directus/directus/assets/9347206/323bbda4-5c07-40fb-8350-572b7a9aa6b4

### To Reproduce

1.) Setup a One-To-Many list
2.) Use a display template to see some fields of the related collection
3.) Update an item

### Directus Version

v10.4.3

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by DanielBiegler** [comment]
I agree that non-devs might be a little confused by seeing that. I guess the question is: How _should_ this look like? Visually marking them with a different color could be good? Other opinions?

**Comment by rijkvanzanten** [comment]
> I guess the question is: How should this look like? Visually marking them with a different color could be good? Other opinions?

It should ideally match the configured display template as closely as possible, by merging the changes on top of the existing data and handling the relational update structure appropriately 👍🏻

**Comment by AlexGaillard** [comment]
Closing this out as I believe it was fixed a while ago with https://github.com/directus/directus/pull/24232 😃
